docs/dev/building/Options.rst

```diff
@@ -0,0 +1,152 @@
+.. _build-options:
+
+#############
+Build Options
+#############
+
+.. contents:: Typical Options
+  :local:
+  :depth: 1
+
+There are a variety of other settings which you can find in CMakeCache.txt in
+your build folder or by running ``ccmake`` (or another CMake GUI). Most
+DFHack-specific settings begin with ``BUILD_`` and control which parts of DFHack
+are built.
+
+Typical usage may look like::
+
+    # Plugin development with updated documentation
+    cmake ./ -G Ninja -B builds/debug-info/ -DCMAKE_INSTALL_PREFIX=<path to DF> -DCMAKE_BUILD_TYPE:string=RelWithDebInfo -DBUILD_DOCS:bool=ON -DBUILD_PLUGINS=1
```

Comment:
I wouldn't call this "typical usage" with the custom build dir specified like that, but this is probably still legal usage.